pkg/operator/ceph/object/rgw-probe.sh

```diff
@@ -17,11 +17,12 @@ RGW_URL="$PROBE_PROTOCOL://0.0.0.0:$PROBE_PORT"
 
 function check() {
   local URL="$1"
+  # --max-time 3 - override the default 2 minutes wait time down to 3 seconds
   # --insecure - don't validate ssl if using secure port only
   # --silent - don't output progress info
   # --output /dev/stderr - output HTML header to stdout (good for debugging)
   # --write-out '%{response_code}' - print the HTTP response code to stdout
-  curl --insecure --silent --output /dev/stderr --write-out '%{response_code}' "$URL"
+  curl --max-time 3 --insecure --silent --output /dev/stderr --write-out '%{response_code}' "$URL"
```

Comment:
why 3 sec?

Reply:
the healthcheck itself has default `timeoutSeconds: 5`. 3 seconds is enough to detect the network problem and stop curl process right away + some more secs for bash script to process the script when the node is not responsive.

Reply:
The bash script itself should be able to process the curl result in under a second.